kubernetes/zenko/templates/maintenance/debug/kafkaclient.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if .Values.maintenance.debug -}}
+apiVersion: v1
+kind: Pod
+metadata:
+  name: {{ template "zenko.fullname" . }}-debug-kafka-client
+spec:
+  containers:
+  - name: kafka
+    image: solsson/kafka:0.11.0.0
```

Comment:
Thinking out loud: could there be a case where we need an image pull policy for this pod? Wondering if, as kafka releases/updates CLI tools, we could take advantage of them by using the latest image rather than keeping it at v `0.11.0.0`.

Reply:
That's a really fair point. I'll make it configurable.